spec/features/female_missing_info_feature_spec.rb

```diff
@@ -119,4 +126,39 @@
       wait_for { page.current_path == missing_information_prison_prisoners_path(prison.code) }
     end
   end
+
+  context 'when multiple tabs are opened', :js do
+    let(:complexity) { 'medium' }
```

Comment:
given the change, I think this test should have complexity nil so that both screens are filled out both times...